test/integration/managedzone_controller_test.go

```diff
@@ -0,0 +1,91 @@
+//go:build integration
+
+// /*
+// Copyright 2023 The MultiCluster Traffic Controller Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// */
+
+package integration
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/Kuadrant/multicluster-gateway-controller/pkg/apis/v1alpha1"
+	//+kubebuilder:scaffold:imports
+)
+
+var _ = Describe("ManagedZoneReconciler", func() {
+	Context("testing ManagedZone controller", func() {
```

Comment:
I'm guessing this is a copy/paste of the original file with the Before/AfterSuite removed?
No other changes?